app/teachingCall/teachingCallStatus/controllers/TeachingCallStatusCtrl.js

```diff
@@ -25,6 +25,11 @@ class TeachingCallStatusCtrl {
 		$rootScope.$on('teachingCallStatusStateChanged', function (event, data) {
 			$scope.view.state = data;
 			console.log(data);
+
+			TeachingCallStatusService.getCourses($scope.workgroupId, $scope.year)
+				.then(function (res) {
+					res.courses.length === 0 ? $scope.view.state.isCourseTableEmpty = true : $scope.view.state.isCourseTableEmpty = false;
```

Comment:
Why is the variable called `isCourseTableEmpty`? Shouldn't it be something like `scheduleHasCourses` ? Where's the "table" that's referenced?

Reply:
What about:

`$scope.view.state.isCourseTableEmpty = (res.courses.length === 0);`

?